CHANGELOG-1.1.md

```diff
@@ -2,14 +2,17 @@
 
 This is the second release candidate of `v1.1.0`, which focuses on the usability, extensibility and security of TiDB Operator. While we encourage usage in non-critical environments, it is **NOT** recommended to use this version in critical environments.
 
+## Action Required
```

Comment:
`Action Required` seems not correct in this case, maybe we can use 'Upgrade Notes`?